packages/faustwp-core/src/getWordPressProps.tsx

```diff
@@ -67,7 +67,7 @@ export async function getWordPressProps(
       resolvedUrl = '/';
     }
   } else {
-    resolvedUrl = ctx.req.url;
+    resolvedUrl = ctx.resolvedUrl;
```

Comment:
Wouldn't this set the `resolvedUrl` in SSR mode to `null`?

Reply:
I haven't seen null or undefined in SSR here but I've seen `ctx.req.url` to be undefined before.

Reply:
```
resolvedUrl: A normalized version of the request URL that strips the _next/data prefix for client transitions and includes original query values.
```
https://nextjs.org/docs/api-reference/data-fetching/get-server-side-props#context-parameter